stable/node-local-dns/templates/daemonset.yaml

```diff
@@ -50,7 +50,15 @@ spec:
         {{- with .Values.resources }}
         resources: {{- toYaml . | nindent 10 }}
         {{- end }}
-        args: [ "-localip", "{{ .Values.config.localDns }},{{ .Values.config.dnsServer }}", "-conf", "/etc/Corefile", "-upstreamsvc", "{{ include "node-local-dns.fullname" . }}-upstream" ]
+        args:
+        {{- range .Values.configArgs }}
+        - {{ . }}
+        {{- end }}
+        - "{{ .Values.config.localDns }},{{ .Values.config.dnsServer }}"
+        - "{{ include "node-local-dns.fullname" . }}-upstream"
+        - -skipteardown={{ .Values.config.skipTeardown }}
+        - -setupinterface={{ .Values.config.setupInterface }}
+        - -setupiptables={{ .Values.config.setupIptables }}
```

Comment:
This is change you are re-ordering all the args and it looks like the order matters, no?

Before: `-localip 169.254.20.25,172.20.0.10 -conf /etc/Corefile -upstreamsvc node-local-dns-upstream`

After: `-localip -conf /etc/Corefile -upstreamsvc 169.254.20.25,172.20.0.10 node-local-dns-upstream...`